deployment-aws/puppet/mevapi/manifests/nginx.pp

```diff
@@ -51,6 +51,22 @@
         location_alias => "${mevapi::django::static_root}/",
         index_files    => [],
       },
+      'nf-status-update-deny' => {
+        location       => '/api/nextflow/status-update/',
```

Comment:
Might be a good idea to factor this out as a variable to avoid duplication and for security reasons: if a URL changes then it should be updated in both places to keep the restrictions in place.

Reply:
good point. fixed locally, but unsure if we can make it one single variable to apply there and also in django urls.py. Not sure if that's possible, but likely lower priority.

Reply:
This is fine.